cmake/configs/default.cmake

```diff
@@ -528,6 +528,7 @@ hunter_default_version(uuid VERSION 1.0.3)
 hunter_default_version(v8 VERSION 7.4.98-p3)
 hunter_default_version(vectorial VERSION 0.0.0-ae7dc88-p0)
 hunter_default_version(vorbis VERSION 1.3.6-p1)
+hunter_default_version(wavm VERSION 1.0.0)
```

Comment:
don't look like real wavm version

Reply:
Fair enough. However, the real WAVM version is 0.0.0 (https://github.com/WAVM/WAVM/blob/master/VERSION) so I preferred to leave the default 1.0.0 for the hunterized version.